interfaces/builtin/ofono.go

```diff
@@ -340,14 +326,18 @@ func (iface *ofonoInterface) DBusPermanentSlot(spec *dbus.Specification, slot *i
 }
 
 func (iface *ofonoInterface) UDevPermanentSlot(spec *udev.Specification, slot *interfaces.Slot) error {
-	old := "###CONNECTED_SECURITY_TAGS###"
-	udevRule := ofonoPermanentSlotUDev
-	for appName := range slot.Apps {
-		tag := udevSnapSecurityName(slot.Snap.Name(), appName)
-		udevRule += strings.Replace(ofonoPermanentSlotUDevTag, old, tag, -1)
-		spec.AddSnippet(udevRule)
-	}
-	spec.AddSnippet(udevRule)
+	spec.AddSnippet(ofonoPermanentSlotUDev)
+	/*
+	   1.Linux modem drivers set up the modem device /dev/modem as a symbolic link
+	     to the actual device to /dev/ttyS*
+	   2./dev/socket/rild is just a socket, not device node created by rild daemon.
+	     Similar case for chnlat*.
+	   So we intetionally skipped modem, rild and chnlat.
+	*/
+	spec.TagDevice(`KERNEL=="tty[A-Z]*[0-9]*|cdc-wdm[0-9]*"`)
+	spec.TagDevice(`KERNEL=="tun"`)
+	spec.TagDevice(`KERNEL=="tun[0-9]*"`)
```

Comment:
Oh, this `tun[0-9]*` tag is actually an error. I realize this PR didn't introduce this, but we should fix it. See https://github.com/snapcore/snapd/pull/4031

Reply:
Dropped